.orchestra/config/components/bzip2.yml

```diff
@@ -0,0 +1,63 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+#@ load("/lib/make.lib.yml", "make")
+
+#@ bzip2_ver = "1.0.8"
+#@ source_url = "https://sourceware.org/pub/bzip2/bzip2-{}.tar.gz".format(bzip2_ver)
+
+#@yaml/text-templated-strings
+---
+#@ def bzip2_args():
+license: LICENSE
+configure: |
+  mkdir -p "$BUILD_DIR"
+  extract.sh --into "$BUILD_DIR" (@= source_url @)
+
+  sed 's/-Winline//g' -i "$BUILD_DIR"/Makefile*
+  
+install: |
+  cd "$BUILD_DIR"
+  
+  # Build
+  (@= make @) -f Makefile-libbz2_so CC="${CC:-gcc} ${CFLAGS:-} ${CPPFLAGS:-} ${LDFLAGS:-}" PREFIX=/
+  (@= make @) bzip2 bzip2recover CC="${CC:-gcc} ${CFLAGS:-} ${CPPFLAGS:-} ${LDFLAGS:-}" PREFIX=/
```

Comment:
Drop `CC=`. We have our magic magic. We don't need those.

Reply:
The bzip2 makefile disregards those variables, though. Does the magic™ handle it?

Reply:
Our magic uses a compiler wrapper that use a set of custom environment variables to bypass all the build systems and have homogeneous compiler flags in all places.